migrations/20220804162029-create-site-meal-type.js

```diff
@@ -25,6 +39,11 @@ module.exports = {
     });
     // set starting id to larger value so it doesn't conflict with test fixtures
     await queryInterface.sequelize.query('ALTER SEQUENCE "SiteMealTypes_id_seq" RESTART WITH 100;');
+    // create a unique compound index to prevent duplicates
+    await queryInterface.addIndex('SiteMealTypes', {
+      fields: ['SiteId', 'MealTypeId'],
+      unique: true,
+    });
```

Comment:
This "unique" index on the combination of SiteId, MealTypeId makes sure that duplicate records don't get added.